src/sentry/rules/processing/delayed_processing.py

```diff
@@ -45,6 +47,7 @@
 logger = logging.getLogger("sentry.rules.delayed_processing")
 EVENT_LIMIT = 100
 COMPARISON_INTERVALS_VALUES = {k: v[1] for k, v in COMPARISON_INTERVALS.items()}
+CHUNK_BATCH_SIZE = 10000
```

Comment:
It could be interesting to push this higher, or even make it configurable via flagpole (that would also allow us to customize it per project if that was every necessary)

Reply:
👍 makes sense to set via a configuration!

any chance you know of an example of setting a project scope'd int to do that? (i was reading the docs and got a little 😵, seems like it's something like...
```
default_manager.add('projects:delayed-processing-chunk-size', ProjectFeature, FeatureHandlerStrategy.FLAGPOLE)
```

then to set the value i add it to [flagpole.yml](https://github.com/getsentry/sentry-options-automator/blob/main/options/default/flagpole.yml#L735) or is that just to define the conditions when to change the value?

Reply:
Hmm... actually I'm not sure how you'd do this via flagpole right now, I guess it'd just be a raw options automator thing? Since flagpole is just for verifying whether a feature is enabled.

Not a huge deal either way, although maybe we should just make this configurable via options automator with a default, and not worry about being able to customize per project.

Reply:
+1 to raw option :)